examples/compressor.rs

```diff
@@ -0,0 +1,45 @@
+use std::fs::File;
+use web_audio_api::context::{AudioContext, BaseAudioContext};
+use web_audio_api::node::{AudioNode, AudioScheduledSourceNode, DynamicsCompressorNode};
+
+fn main() {
+    env_logger::init();
+
+    let context = AudioContext::default();
+
+    let file = File::open("samples/sample.wav").unwrap();
```

Comment:
This may not be the most interesting file for a compressor..

Reply:
Maybe `think-stereo-48000.wav` is already a bit more interesting

Reply:
I was afraid this file was already hardcore compressed but we can actually ear the difference. I update that